README.md

```diff
@@ -1,56 +1,86 @@
-* [Jenkins](https://jenkins-ci.org/)
-* [MongoDB](https://www.mongodb.org/)
+* [Node.js](https://nodejs.org/en/) backend
+  ([ES2015 syntax](https://git.io/es6features) via [Babel](https://babeljs.io))
+* [GitHub OAuth API](https://developer.github.com/v3/oauth/) via
+  [Passport](https://github.com/jaredhanson/passport-github)
+* [MongoDB](https://www.mongodb.org/) with
+  [Mongoose](https://github.com/Automattic/mongoose)
 * [Stripe Connect](https://stripe.com/connect)
-* [Liftoff](https://github.com/elementary/liftoff) (Cody's build scripts, repo currently private)
-* **[Learn more](https://docs.google.com/document/d/1nHCnxNpaQI8G2VdJKFeri12krLpgtUQllMj8_PdZ7P8/edit)**
-
-Houston is the glue between GitHub and our repository system. Ideally when you first create a project on GitHub, you would go to houston and initialize the repository. This will setup houston to monitor the repository with GitHub hooks (eventually, for now it's manual with houston's web interface).
-
-When a release is created on GitHub, and you choose to publish that release through Houston, Houston will run some tests on it. These tests currently include automatically checking for an app icon, and some strings in the .desktop file. It will be expanded in the future. At the same time, the repository will be passed to Jenkins, which will build the repo into a deb package. After it's built, a person from elementary will install the application and test for a multitude of things including but not limited to performance, aesthetics, and nativeness.
+* [Liftoff](https://github.com/elementary/liftoff)
+  (Cody's build scripts, repo currently private)
+
+Houston is the glue between GitHub and our repository system. Ideally when you
+first create a project on GitHub, you would go to Houston and initialize the
+repository. This will setup Houston to monitor the repository with GitHub hooks.
+
+When a release is created on GitHub, you will get an option to publish that
+release in Houston's dashboard. Houston will then clone the repository, run
+tests on the code, build the code with
+[Liftoff](https://github.com/elementary/liftoff), and publish it to a
+**testing** repository. At this point someone from elementary will install the
+application and continue running tests. This ensures the best performance, user
+experience, and desktop integration for any application in AppHub.
+
+If any test fails, Houston will submit an issue on GitHub with detailed
+information on the problems, and in some cases, solutions to fix it. After
+fixing these issues you will be able to resubmit your application for
+publishing.
+
+At this point, the approved application will be moved into the stable repo, and
+available to all users of elementary. We will be using
+[aptly](https://github.com/smira/aptly) behind nginx to host the repository.
+Ultimately this repository will be the focus of
+[AppCenter](https://launchpad.net/appcenter) and elementary OS's 3rd party
```

Comment:
Should `elementary OS's` be `the elementary OS`? I'm undecided, but "OS's" can be kind of awkward.